Tests/iOS/PinSafeAreaTests.swift

```diff
@@ -415,8 +408,8 @@ class PinSafeAreaMoreTestsSpec: QuickSpec {
                 setupWindow(with: navigationController)
 
                 // MATCH safeAreaInsets!
-                expect(mainView.safeAreaInsetsDidChangeCalledCount).to(equal(expectedSafeAreaInsetsDidChangeCalledCount))
-                expect(mainView.subView.safeAreaInsetsDidChangeCalledCount).to(equal(expectedSubViewSafeAreaInsetsDidChangeCalledCount))
+                expect(mainView.safeAreaInsetsDidChangeCalledCount) > 0
```

Comment:
We could also update all other similar cases in safeArea unit tests.

Reply:
Will do :)